orttraining/orttraining/core/graph/gradient_builder_base.h

```diff
@@ -85,33 +87,57 @@ class GradientBuilderBase {
     return gradient_graph_config_;
   }
 
+  void TryRecordStashedForwardGraphTensor(const std::string& name) const {
+    if (stashed_tensors_.find(name) == stashed_tensors_.end()) {
+      stashed_tensors_.insert(name);
```

Comment:
>stashed_tensors_ [](http://example.com/codeflow?start=6&length=16)

stashed_tensors_ is a unordered_set, ok to insert existing value?

Reply:
I suppose since it will return the element with the equivalent key.